client/utils/network.py

```diff
@@ -26,28 +22,3 @@ def check_ssl():
     else:
         log.info('SSL module is available')
         return ssl
-
-def api_request(access_token, server, route, insecure=False, arguments={}):
```

Comment:
Note for the future: This could be useful for other protocols - but nothing else uses this so we're going to remove it.